packages/client/package.json

```diff
@@ -32,7 +32,8 @@
     "lint": "eslint \"src/**/*.+(ts|js|tsx)\"",
     "format": "prettier \"src/**/*.{js,ts,tsx}\" --write",
     "format:check": "prettier \"src/**/*.{js,ts,tsx}\" --check",
-    "test": "vitest",
+    "test": "vitest run",
```

Comment:
Vitest run in watch mode by default locally which conflicts with Turborepo's caching, so we need to specify separate commands.